polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -419,6 +434,14 @@ const navTransitionClasses = {
 export function Frame(props: FrameProps) {
   const i18n = useI18n();
   const mediaQuery = useMediaQuery();
-
-  return <FrameInner {...props} i18n={i18n} mediaQuery={mediaQuery} />;
+  const features = useFeatures();
```

Comment:
I think adding the useFeatures hook here is causing tests to fail. We might be able to do this at the AppProvider level to fix that. Otherwise I think we'd need to wrap portals in a frame, which we definitely don't want